content/docs/04-clusters/03-edge/02-edge-configuration/35-installer-reference.md

```diff
@@ -155,6 +155,52 @@ Network settings specific to the network interface of the edge host. You can con
 | `networkInterface.gateway` | The network gatway IP address. |
 | `networkInterface.nameserver` | The IP address of the DNS nameserver this interface should route requests to.| 
 
+## User ID (UID) Parameters
```

Comment:
```suggestion
## Device ID (UID) Parameters
```
This is not a user id.

Reply:
Okay, but in that case, shall I remove the UID in the brackets. as Device ID and in brackets, we have UID, it doesn't make sense.